diagnostic.md

```diff
@@ -6,70 +6,74 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+resources :books, only: [:index, :show]
+resources :books, except: [:edit, :view, :destroy, :update, :create]
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+It configures the database that you created to the specifics that you hold in your
+migrate file. i.e what columns and what type of data fields it should have.
```

Comment:
It executes any code found within the Active Record migration files in db/migrate. The migrations make it so that we don't have to modify our database directly.